account_invoice_inter_company/views/account_move_views.xml

```diff
@@ -13,4 +13,30 @@
             </xpath>
         </field>
     </record>
+
+    <record id="view_move_form_inherit" model="ir.ui.view">
+        <field name="name">account_move_view_form_inherit</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <notebook position="inside">
+                <page string="Intercompany Info" name="intercompany_info">
+                    <group>
+                        <group
+                            string="Invoice Info"
+                            attrs="{'invisible': [('auto_invoice_id', '=', False)]}"
+                        >
+                            <field name="auto_invoice_id" />
+                        </group>
+                        <group
+                            string="Bill Info"
+                            attrs="{'invisible': [('related_bill_id', '=', False)]}"
+                        >
+                            <field name="related_bill_id" />
```

Comment:
Is this invoice of other company?

Reply:
yes

Reply:
Then you can't add it, or you will have access errors if you are not logged in both companies.

Reply:
No. You just wont see it.

Reply:
That's not how this works I'm afraid. Have you tried it?

Reply:
https://recordit.co/Na7NKn04Ns

Reply:
Uhm, I didn't know about that curious behavior, but then there's a misuse of the feature: if you are not logged in both companies, you may think that the invoices isn't is an inter-company one. I think you have to put 2 computed fields with `compute_sudo=True`:

- Related invoice number as text.
- Related company.

And you can replace the text by the many2one if the company is in allowed ones.

Reply:
how about this: https://recordit.co/UNYQp7OWns

Reply:
You will have user complaining about such error. And out of this, you don't know the company of the bill. I consider that info important.

Reply:
Company of a bill you can see in the Customer field.

Reply:
Yes, true...

The access problem when you click is something to be fixed anyways.

Reply:
Updated.